bor/README.md

```diff
@@ -0,0 +1,166 @@
+# PRELIMINARY TERMINOLOGY
+
+* A `side-transaction` is a normal heimdall transaction but the data with which the message is composed needs to be voted on by the validators since the data is obscure to the consensus protocol itself and it has no way of validating the data's correctness.
+* A `sprint` comprises of 16 blocks.
```

Comment:
Do you think it's good to mention 16 bor blocks here?

Reply:
Addressed [here](https://github.com/maticnetwork/heimdall/pull/1071/commits/4fc537d495cb854277cacff52c1b4a6282ff0bdf)